modules/planning/tasks/optimizers/path_time_heuristic/gridded_path_time_graph.cc

```diff
@@ -266,7 +266,8 @@ void GriddedPathTimeGraph::GetRowRange(const StGraphPoint& point,
 
   const auto max_s_size = dimension_s_ - 1;
 
-  const double max_acceleration = std::abs(vehicle_param_.max_acceleration());
+  const double max_acceleration = std::abs(
```

Comment:
@JasonXu12 Please do a min comparison with the settling here with max vehicle acceleration in vehicle params. In case the setting here exceeds the vehicle parameter.

Reply:
Good point. Fixed per request.